extensions/ql-vscode/src/vscode-tests/cli-integration/jest-runner-vscode.config.js

```diff
@@ -25,6 +25,7 @@ const config = {
     ...baseConfig.extensionTestsEnv,
     INTEGRATION_TEST_MODE: "true",
   },
+  retries: 3,
```

Comment:
Is there any guarantee that 3 retries will resolve the flakiness?

I know it's a lot to ask after so much effort doing the jest migration, but are we able to pinpoint what the failure reasons are at the moment for windows?